src/content/en/tools/workbox/guides/faq.md

```diff
@@ -0,0 +1,104 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: Frequently Asked Questions when developing with Workbox
+
+{# wf_updated_on: 2018-02-28 #}
+{# wf_published_on: 2018-02-28 #}
+{# wf_blink_components: N/A #}
+
+# FAQ {: .page-title }
+
+## Why isn't my service worker being used? {#sw-scope}
+
+This is a fairly common and easy problem to hit when working with service
+workers.
+
+You'll register the service worker, everything seems to work and files are
+cached, but refreshing the page or going offline doesn't show your files
+(i.e. the browser isn't using the files in the cache).
+
+The reason this occurs is that the "scope" of your service worker doesn't
```

Comment:
You'd see similar symptoms for the first navigation when you are not using `clientsClaim` (and sometimes when you aren't using `skipWaiting`, if you had a previous service worker open in other tabs).

I like the material that already exists about this at https://developers.google.com/web/fundamentals/primers/service-workers/lifecycle#scope_and_control, so I think it could just be linked to here rather than trying to replicate it. But it would be a good idea to acknowledge that there are other possible scenarios to investigate when you're not seeing the right SW in control.

Reply:
I rejigged things and reworded so it highlights *it's one reason this occurs* and I moved how to identify *if* this is the problem early on.